openapi/components/schemas/RunRequest.yaml

```diff
@@ -27,6 +27,16 @@ properties:
     type: object
     additionalProperties:
       type: string
+  workflow_engine:
+    type: string
+    description: >-
+      The workflow engine, must be one supported by this WES instance. Required if workflow_engine_version is provided.
+  workflow_engine_version:
+    type: string
+    description: >-
+      The workflow engine version, must be one supported by this WES instance.
+      If workflow_engine is provided, but workflow_engine_version is not, clients can make no assumptions with regard to the engine version the WES instance uses to process the request if 
+      that WES instance supports multiple versions of the requested engine.
```

Comment:
@vsmalladi can we add a phrase about what happens when the `workflow_engine` and `workflow_engine_version` or not defined? For example, is it up to the engine what the default `workflow_engine/version` or is there a different way to report this?

Reply:
@patmagee Updated.